openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeContour.kt

```diff
@@ -310,10 +337,24 @@ data class ShapeContour(val segments: List<Segment>, val closed: Boolean, val po
         return Matrix44.fromColumnVectors(dx, dy, Vector4.UNIT_Z, dt)
     }
 
+    /**
+     * Recursively subdivides linear [Segment]s to approximate Bézier curves.
+     *
+     * Works like [adaptivePositionsAndCorners] but discards list of booleans.
```

Comment:
discards list of booleans?

Reply:
I can phrase it a little better but essentially it does what it says, because adaptivePositionsAndCorners returns a Pair<List<Vector2>, List<Boolean>>, but adaptivePositions discards the right side of the pair.

Reply:
I see... 
`but doesn't return x`?